### ISSUE TITLE
Fix CPU Graph

### ISSUE THREAD

**Issue by ToBiDi0410** [ToBiDi0410/SpigotDash#7]
With some Configuration, the CPU Usage Graph doesn't Work correctly.

**Comment by nsde** [comment]
![image](https://user-images.githubusercontent.com/67185896/152700175-f7851bbf-dbf9-4bba-8f01-a2ddbf41e454.png)
Works as expected for me.

**Comment by ToBiDi0410** [comment]
Yeah. For most users it does but for some I have no clue why it won't

**Comment by nsde** [comment]
> Yeah. For most users it does but for some I have no clue why it won't

Maybe because of the browser? Or OS? I'm using Firefox & Linux.

**Comment by ToBiDi0410** [comment]
I'm not really sure. It's definitely no Problem with the Frontend (Webinterface).

I had one User that allowed me to test some things on an Configuration where it won't work. But I was not able to track down the Reason. Even if it was the same Java Version, Build Source and OS that I was also using.
